### PR TITLE
Profile modal keyboard panning responder

### PR DESCRIPTION
**Pull request by oidoug**
### Description

This PR simplifies ProfileModal and KeyboardFixedOpenLayout to use the new `keyboardavoidingview` panning behavior.

ProfileModal and components that use it could be rebuilt with `restyle` containers, I started for WalletProfileModal but it's not trivial because there are more components using it.

### Checklist

- [x] All UI changes have been tested on a small device

### Screenshots

Screenshot

Screenshot

Screenshot

### CODE REVIEW

**Review by paulinhapenedo — APPROVED** [review]
Looks good! 👏

File: src/components/layout/KeyboardFixedOpenLayout.js [hunk 1]
```diff
@@ -1,48 +1,18 @@
...
 
-const Container = styled(Transitioning.View)`
-  height: ${({ height }) => height};
-  left: 0;
-  position: ${android ? 'relative' : 'absolute'};
-  right: 0;
-  top: 0;
+const Container = styled(View)`
+  height: 100%;
+  width: 100%;
```
Comment (danibonilha) [inline src/components/layout/KeyboardFixedOpenLayout.js]: nit: Would it be possible to replace this styled view with the restyle Container ?

File: src/components/layout/KeyboardFixedOpenLayout.js [hunk 2]
```diff
@@ -1,48 +1,18 @@
...
   return (
-    <Container height={containerHeight} ref={ref} transition={transition}>
-      <KeyboardAvoidingView behavior="height" enabled={!keyboardHeight}>
-        <InnerWrapper {...props} insets={insets} />
+    <Container>
+      <KeyboardAvoidingView behavior="padding">
+        <Centered {...props} />
       </KeyboardAvoidingView>
     </Container>
   );
```
Comment (danibonilha) [inline src/components/layout/KeyboardFixedOpenLayout.js]: Actually, can we just get rid of this `KeyboardFixedOpenLayout` component ? it's used  only in the `ProfileModal`, which is a pretty small component, couldn't  we just add the KeyboardAvoidingView and the container there ?

File: src/components/layout/KeyboardFixedOpenLayout.js [hunk 3]
```diff
@@ -1,48 +1,18 @@
...
-    // eslint-disable-next-line react-hooks/rules-of-hooks
-    useEffect(() => ref.current.animateNextTransition(), [containerHeight]);
-
+export default function KeyboardFixedOpenLayout({ ...props }) {
   return (
-    <Container height={containerHeight} ref={ref} transition={transition}>
-      <KeyboardAvoidingView behavior="height" enabled={!keyboardHeight}>
-        <InnerWrapper {...props} insets={insets} />
+    <Container>
+      <KeyboardAvoidingView behavior="padding">
```
Comment (danibonilha) [inline src/components/layout/KeyboardFixedOpenLayout.js]: nit: We could use the `Device.keyboardBehavior`

**Review by danibonilha — APPROVED** [review]
Nice fix, I've just added a suggestion to cleanup a bit more

File: src/components/layout/KeyboardFixedOpenLayout.js (hunk 2 above)
Comment (oidoug) [reply src/components/layout/KeyboardFixedOpenLayout.js]: Yes, I tried but was having issues making the centered view stretch in width, some component is messing with the layout. I'll give it another go.

Comment (oidoug) [reply src/components/layout/KeyboardFixedOpenLayout.js]: Agh, I was trying too hard before... went back to it now and found an easy way around it, will push soon.

**Pull request merged**